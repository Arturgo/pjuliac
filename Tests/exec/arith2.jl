
x = 20

println(2x+2)
println(2(x)+2)
println(2(x+1))
println((2)x+2)

println(2+3x)
println(2*3x)
println(2^3x)
println(2(div(x,2)))

