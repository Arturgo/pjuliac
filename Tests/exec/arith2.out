42
42
42
42
62
120
1152921504606846976
20
